src/routes/safe/components/Apps/utils.ts

```diff
@@ -153,8 +153,6 @@ export const isAppManifestValid = (appInfo: SafeApp): boolean =>
   appInfo.name !== 'unknown' &&
   // `description` exists
   !!appInfo.description &&
-  // `url` exists
-  !!appInfo.url &&
```

Comment:
I wouldn't delete this. The parameter will be mandatory later. We just disable it now to check that all apps are correctly adapted

I would add a comment explaining this and adding the url parameters in the tests

Reply:
what would be the purpose of it?
remember that we already have this: "providedBy": {"name": 'Gnosis', "url": 'https://gnosis.io'}

Reply:
Didn't follow the whole discussion in the feature ticket. Will check there